helpdesk/teams/views.py

```diff
@@ -116,28 +115,28 @@ class TeamDetailTimelineView(LoginRequiredMixin, DetailView):
     def get_entries(self) -> QuerySet[Any]:
         fields = ("entry_type", "entry_timestamp", "entry_user", "entry_content", "entry_style_info")
 
-        tickets = Ticket.objects.filter(team=self.object).order_by().annotate(
+        ticket_opens = TicketEvent.objects.filter(ticket__team=self.object, new_status__exact="OP").annotate(
             entry_type=Value('Ticket Opened ', output_field=CharField()),
             entry_timestamp=F("created_at"),
-            entry_user=F("opened_by"),
-            entry_content=F("description"),
+            entry_user=F("user"),
+            entry_content=F("comment"),
             entry_style_info=Value('is-info', output_field=CharField()),
         ).values(*fields)
 
-        comments = TicketComment.objects.filter(ticket__team=self.object).order_by().annotate(
-            entry_type=Value('Ticket Comment', output_field=CharField()),
+        ticket_resolves = TicketEvent.objects.filter(ticket__team=self.object, new_status__exact="RS").annotate(
+            entry_type=Value('Ticket Resolved ', output_field=CharField()),
             entry_timestamp=F("created_at"),
-            entry_user=F("author"),
-            entry_content=F("content"),
-            entry_style_info=Value('', output_field=CharField()),
+            entry_user=F("user"),
+            entry_content=F("comment"),
+            entry_style_info=Value('is-success', output_field=CharField()),
         ).values(*fields)
```

Comment:
Kinda nasty, but it avoids doing IFs inside the SQL, and is probably much easier to understand